test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
**[REQUIRED]** According to the project requirements, this file should be ignored by git. Please include this file in the `.gitignore` file

Reply:
I have already added test.md in .gitignore. Thank you.